_includes/install_command.html

```diff
@@ -0,0 +1,24 @@
+<div class="inline">
+    {% highlight bash %} $ {{ include.code }} {% endhighlight %}
+
+    <button
+        id="copyButton"
+        aria-label="Copy to clipboard"
+        style="display: none"
+        onclick="copyToClipboard()"
+    >
+        📋
+    </button>
+</div>
+
+<script>
+    function copyToClipboard() {
+        navigator.clipboard.writeText(`{{include.code}}`);
+    }
+
+    if (navigator.clipboard) {
+        document
+            .getElementById("copyButton")
+            .style.removeProperty("display");
+    }
```

Comment:
Sorry to keep changing my mind on this but: I think it'd be fine to use that library you originally proposed for this if it made this easier: how about this code instead:

- is in a seperate `.js` file
- if `navigator.clipboard` exists: looks for all elements with class (not ID) `copyable` 
- writes a clipboard button to the right of those elements
- adds the logic to copy the contents of the current element's text (rather than passing it through and duplicating it in Jekyll)

Reply:
Yeah, I've done something similar.

Don't think we need the other library.